superset/models/core.py

```diff
@@ -102,7 +102,7 @@ class KeyValue(Model):  # pylint: disable=too-few-public-methods
     value = Column(utils.MediumText(), nullable=False)
 
 
-class CssTemplate(Model, AuditMixinNullable):
+class CssTemplate(Model, AuditMixinNullable, UUIDMixin):
```

Comment:
This was already incorrect, but in general it's idiomatic to place the mixin _before_ the main class to ensure MRO (otherwise functionality in the main class will overwrite functionality in the mixin):
```suggestion
class CssTemplate(AuditMixinNullable, UUIDMixin, Model):
```